content/paths/events__get_events.yml

```diff
@@ -146,6 +146,7 @@ parameters:
           - METADATA_TEMPLATE_UPDATE
           - MOVE
           - NEW_USER
+          - OAUTH2_ACCESS_TOKEN_REVOKE
```

Comment:
Correct me if I am wrong, but this event is only in the `admin_logs` right? If so we also need to update the relevant guide.